sdk/python/.openapi-generator/FILES

```diff
@@ -41,6 +41,7 @@ pulumi_esc_sdk/models/update_environment_tag.py
 pulumi_esc_sdk/models/update_environment_tag_current_tag.py
 pulumi_esc_sdk/models/update_environment_tag_new_tag.py
 pulumi_esc_sdk/models/value.py
+pulumi_esc_sdk/pulumi-plugin.json
```

Comment:
This file is generated, you cannot modify it directly. Modify the file in templates so this is the generated result.